compat/README.md

```diff
@@ -1,3 +1,15 @@
 # Source Compatibility Suite
 
+## Running
```

Comment:
Could you briefly explain what the compatibility suite do?

Reply:
Added above, also see #226